x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/autoscaling/MlAutoscalingDeciderService.java

```diff
@@ -922,52 +942,54 @@ Optional<AutoscalingDeciderResult> checkForScaleUp(
         return Optional.empty();
     }
 
-    // This calculates the the following the potential future free capacity
-    // Since jobs with lookback only datafeeds, and data frame analytics jobs all have some potential future end date
-    // we can assume (without user intervention) that these will eventually stop and free their currently occupied resources.
-    //
-    // The capacity is as follows:
-    // tier: The sum total of the resources that will be eventually be available
-    // node: The largest block of memory that will be free on a given node.
-    // - If > 1 "batch" ml tasks are running on the same node, we sum their resources.
+    /**
+     * This calculates the the following the potential future free capacity
```

Comment:
Simplify this sentence? Perhaps just `This calculates the potential future free capacity.`